api/test/integration/conftest.py

```diff
@@ -80,90 +80,76 @@ def pytest_tavern_beta_before_every_test_run(test_dict, variables):
     variables["test_login_token"] = get_token_login_api()
 
 
-def build_and_up(env_mode: str, interval: int = 10, interval_build_env: int = 10,
-                 build: bool = True) -> dict:
+def build_and_up(env_mode: str, interval: int = 10, build: bool = True):
     """Build all Docker environments needed for the current test.
 
     Parameters
     ----------
     env_mode : str
         Indicates the environment to be used in the process.
     interval : int
-        Time interval between every healthcheck.
-    interval_build_env : int
-        Time interval between every docker environment healthcheck.
+        Time interval between every build.
     build : bool
         Flag to indicate if images need to be built.
-
-    Returns
-    -------
-    dict
-        Dict with healthchecks parameters.
     """
-    os.chdir(env_path)
-    values = {
-        'interval': interval,
-        'max_retries': 90,
-        'retries': 0
-    }
-    values_build_env = {
-        'interval': interval_build_env,
-        'max_retries': 3,
-        'retries': 0
-    }
-   # Get current branch or tag
-    with open('../../../../.git/HEAD', 'r') as f:
+    # Get current branch or tag
+    with open('../../../.git/HEAD', 'r') as f:
         ref = f.readline().strip()
+
     if ref.startswith("ref:"):
         current_branch = ref.split("refs/heads/")[1]
     else:
         current_branch = ref.split("/")[-1]
-        
-    os.makedirs(test_logs_path, exist_ok=True)
+
+    if build:
+        # Ping the current branch tarball used to build the manager image.
+        response = requests.get(f"https://github.com/wazuh/wazuh/tarball/{current_branch}")
+        if response.status_code == 404:
+            pytest.fail("Current branch tarball doesn't exist")
```

Comment:
We should include a condition for when the response code is not successful along with a message describing it.